scala3/scaladoc.md

```diff
@@ -0,0 +1,74 @@
+
+For more information how to use full power of docstings, check out [Scaladoc docstrings][scaladoc-docstrings]
+
+
+## Static site
+
+Scaladoc brings easy way for creating static sites for both documentation and blog posts in the similar way as Jekyll does.
+Thanks to that feature you can store your documentation along-side with Scaladoc API docs output in a very convenient way.
+
+For more info about how to configure static site documentation check out [Static documentation][static-documentation] chapter
+
+![](../resources/images/scala3/scaladoc/static-site.png)
+
+## Blog posts
+
+Blog posts are a specific type of static site. To find more information about them look at the dedicated page for [Built-in blog][built-in-blog]
+
+![](../resources/images/scala3/scaladoc/blog-post.png)
+
+## Social links
+
+Scaladoc provides an easy way to configure your social media links e. g. twitter or gitter. For more information look at the subsection of 
+[Scaladoc settings][social-links] chapter.
+
+![](../resources/images/scala3/scaladoc/social-links.png){: style="width: 180px"}
+
+# Experimental features
+
+These features are not stable to be released with scaladoc, however we are happy to hear your feedback. Each feature has its own thread at scala-lang contributors site, where you can share your opinions.
+
+## Snippets compiler
+
+One of the experimental features of Scaladoc will be snippets compiler. This tool will allow you to compile snippets that you attach to your docstring 
+to check that they actually behave as intended, e. g. compile or throw some exception. The feature is very similar to `tut` or `mdoc` tools, 
+but will be shipped with Scaladoc out of the box for easy setup. Making snippets interactive, e. g. user could edit them and compile in the browser is taken into account, though it is not in scope yet.
+
+For more information you can follow this [thread](https://contributors.scala-lang.org/t/snippet-validation-in-scaladoc-for-scala-3/4976)
+
+![](../resources/images/scala3/scaladoc/snippet-compiler2.gif)
+![](../resources/images/scala3/scaladoc/snippet-compiler1.gif)
+
+## Type-signature browsing
+
+Haskell programmers are probably familiar with Hoogle - documentation search engine that allows you browse the docs by signatures rather than symbolic 
+names. Since many Scala developers are also functional programming fans, we decided to add similar functionality for the Scaladoc. The whole tool is based on [Inkuire](https://github.com/VirtusLab/Inkuire) which works with Kotlin and is being currently adapted to fit Scala3 type system. 
+
+![](../resources/images/scala3/scaladoc/inkuire.gif)
+
+To be up-to-date with this feature you can follow this [thread](https://contributors.scala-lang.org/t/pre-sip-scaladoc-search-by-type-signature/1604/15).
+Note that this is very basic demo of this feature. We still don't support type lambdas, higher-kinded types and many more scala features.
```

Comment:
```suggestion
Haskell programmers are probably familiar with Hoogle - a documentation search engine that allows you browse the docs by signatures rather than symbolic 
names. Since many Scala developers are also functional programming fans, we decided to add a similar functionality to Scaladoc. The whole tool is based on [Inkuire](https://github.com/VirtusLab/Inkuire), which works with Kotlin and is being currently adapted to Scala 3. 

![](../resources/images/scala3/scaladoc/inkuire.gif)

To be up-to-date with this feature you can follow this [thread](https://contributors.scala-lang.org/t/pre-sip-scaladoc-search-by-type-signature/1604/15).
Note that this is very basic demo of this feature. We still don't support type lambdas, higher-kinded types, and many more Scala features.
```

Reply:
Fixed